src/generic/datavgen.cpp

```diff
@@ -1724,6 +1822,7 @@ wxDataViewMainWindow::wxDataViewMainWindow( wxDataViewCtrl *parent, wxWindowID i
     m_useCellFocus = false;
     m_currentRow = (unsigned)-1;
     m_lineHeight = GetDefaultRowHeight();
+    m_rowHeightCache = new HeightCache();
```

Comment:
This should only be allocated and, most importantly, used if/when necessary, i.e. when the control does have `wxDV_VARIABLE_LINE_HEIGHT` style. We definitely don't want to have all this (completely unnecessary) overhead for fixed height controls.